package.json

```diff
@@ -12,11 +12,14 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "downloadjs": "^1.4.7",
+    "html2canvas": "^1.4.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.14.2"
   },
   "devDependencies": {
+    "@types/downloadjs": "^1.4.3",
```

Comment:
I think you don't need these anymore! _Don't_ just delete these lines, you need to do `npm uninstall ...`